sourcemash/frontend/templates/splash.tpl

```diff
@@ -0,0 +1,6 @@
+<div class="row">
+  <div class="col s12 valign-wrapper">
+    <img src="/static/solologo.svg">
+    <img src="/static/logotext.svg">
+  </div>
+</div>
```

Comment:
KO just used MS Paint. I made the SVG files from the EPS files (which are also transparent)